app/models/episode.rb

```diff
@@ -224,15 +222,11 @@ def overrides
   end
 
   def categories
-    self[:categories] ||= []
+    self[:categories] || []
```

Comment:
I opted to leave this postgres array field nil in the database, instead of an empty array.  Assuming it's up to the SQL-er to check for `NOT NULL` when I get around to querying on this.